### PR TITLE
Update EMML1292.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New manuscript, necessary for description and study of further sewasew-manuscripts.

### CODE REVIEW

File: EMML/1001-2000/EMML1292.xml [hunk 1]
```diff
@@ -32,40 +32,981 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0004HMML"/>
-                  <collection>Addis Ababa University. YaʼItyop̣yā ṭenātenā meremer taqwām, Addis Ababa</collection>
+                  <collection>Addis Ababa University.
```
Comment (eu-genia) [inline EMML/1001-2000/EMML1292.xml:35]: please change repository and collection as here in Example 2 
https://betamasaheft.eu/Guidelines/?q=EMML&id=identifiersStructure

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: EMML/1001-2000/EMML1292.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply EMML/1001-2000/EMML1292.xml:35]: ok.

File: EMML/1001-2000/EMML1292.xml [hunk 2]
```diff
@@ -31,20 +31,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
          <sourceDesc>
             <msDesc xml:id="ms">
                <msIdentifier>
-                  <repository ref="INS0004HMML"/>
-                  <collection>Addis Ababa University.
-                     YaʼItyop̣yā ṭenātenā meremer taqwām, Addis Ababa</collection>
+                  <repository ref="INS0002IHA">Institute of Ethiopian Studies</repository>
```
Comment (eu-genia) [inline EMML/1001-2000/EMML1292.xml:34]: IES has ID INS0501IES
(all IDs that have IHA in them belong to the IslHornAfr project and should not be used by BM)

File: EMML/1001-2000/EMML1292.xml [hunk 3]
```diff
@@ -31,20 +31,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+                  <collection ref="INS0004HMML">Ethiopian Manuscript Microfilm Library</collection>
                   <idno>MS EMML no. 1292</idno>
                   <idno>MS EAP286/1/1/6</idno>
```
Comment (eu-genia) [inline EMML/1001-2000/EMML1292.xml:37]: these should be altIdentifier

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: EMML/1001-2000/EMML1292.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply EMML/1001-2000/EMML1292.xml:40]: ok

Comment (eu-genia) [reply EMML/1001-2000/EMML1292.xml:40]: the first is not alt )

Comment (DenisNosnitsin1970) [reply EMML/1001-2000/EMML1292.xml:40]: Oh sorry...

**Review by eu-genia — APPROVED** [review]

**Pull request merged**